antarest/study/storage/variantstudy/model/command/update_st_storages.py

```diff
@@ -44,6 +45,13 @@ class UpdateSTStorages(ICommand):
 
     storage_properties: STStorageUpdates
 
+    @model_validator(mode="after")
+    def validate_properties_against_version(self) -> "UpdateSTStorages":
```

Comment:
Minor: you can return typing.Self to not have the class name between quotes which in my opinion looks weird in general